File: mobile/src/main/java/treehou/se/habit/ui/colorpicker/LightContract.java
package treehou.se.habit.ui.colorpicker;


import se.treehou.ng.ohcommunicator.connector.models.OHItem;
import treehou.se.habit.mvp.BasePresenter;
import treehou.se.habit.mvp.BaseView;

public class LightContract {

    interface View extends BaseView<Presenter> {

    }

    interface Presenter extends BasePresenter {
        void setHSV(OHItem item, int hue, int saturation, int value);
    }
}

File: mobile/src/main/java/treehou/se/habit/ui/colorpicker/LightContract.kt
package treehou.se.habit.ui.colorpicker


import se.treehou.ng.ohcommunicator.connector.models.OHItem
import treehou.se.habit.mvp.BasePresenter
import treehou.se.habit.mvp.BaseView

class LightContract {

    interface View : BaseView<Presenter>

    interface Presenter : BasePresenter {
        fun setHSV(item: OHItem, hue: Int, saturation: Int, value: Int)
    }
}
